.mailmap

```diff
@@ -498,3 +498,4 @@ Zhi Zhang <willzzhang@tencent.com>
 Zhi Zhang <willzzhang@tencent.com> <zhangz.david@outlook.com>
 Zhi Zhang <willzzhang@tencent.com> <zhangzhi@localhost.localdomain>
 Zhuang Xiaochun <zhuangxc89@163.com>
+Zhang Lei <243290414@qq.com> <zhanglei@trendytech.com.cn> 
```

Comment:
@theanalyst 
No. You asked to reverse the order of emails in the above comment. But conventionally, the first email ID added in .mailmap is included in the .organizationmap. It is followed in other people's entry as well. I'm not sure if this makes any technical issues though. But that is what being followed. Please check by yourself.